docs/index.rst

```diff
@@ -14,11 +18,32 @@ Welcome to Earthpy's documentation!
    spatial-raster
    spatial-vector
 
-
-
-Indices and tables
+Indices and Tables
 ==================
 
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+
+About Earthpy's Documents
```

Comment:
Just an FYI -- this line is also why the build is failing
see [here](https://travis-ci.org/earthlab/earthpy/builds/448842865#L2072)

In `rST` you need to have the heading symbols line up / have the same number of characters as the heading text. Because I am asking you to remove this entire section the build should be OK after you do that.